src/JuMPContainer.jl

```diff
@@ -239,6 +269,13 @@ Base.values(d::JuMPDict)  = values(d.tupledict)
 Base.keys(d::JuMPArray)   = KeyIterator(d)
 Base.values(d::JuMPArray) = ValueIterator(d.innerArray)
 
+# TODO: add keys/values for Array
+Base.keys(d::Array) = CartesianRange(size(d))
+Base.start(ci::CartesianIndex)    = start(ci.I)
+Base.next( ci::CartesianIndex, k) = next(ci.I, k)
+Base.done( ci::CartesianIndex, k) = done(ci.I, k)
+Base.values(d::Array) = d
```

Comment:
These methods would probably be more appropriate in ``Base``, if deemed appropriate.